runtime/caml/misc.h

```diff
@@ -444,6 +444,10 @@ extern int caml_snwprintf(wchar_t * buf,
 #  endif
 #endif
 
+#ifdef NAKED_POINTERS_CHECKER
+extern int caml_naked_pointers_detected;
+#endif
```

Comment:
Why declare this variable in `<caml/misc.h>` ?  This header is already a dumping ground.

Reply:
I think I got cold feet for some reason about putting it in `major_gc.c`... I've moved it back there.